docs/source/installation.mdx

```diff
@@ -25,7 +25,6 @@ appropriate extras group.
 
 | extra name   | description                                                               |
 |--------------|---------------------------------------------------------------------------|
-| accelerate   | To use accelerate for model and data parallelism with transformers models |
```

Comment:
We'll want to offer a no parallelism option I think

Reply:
not sure what you mean, parallelism is not enabled by default, we have to set `model_parallel=True`

Reply:
Using accelerate is mandatory to launch lighteval, but we might want a setup with no DP/MP/PP of any kind, without gpu use